tests/typescript-mock-data.spec.ts

```diff
@@ -389,3 +389,10 @@ it('should preserve underscores if transformUnderscore is false', async () => {
     );
     expect(result).toMatchSnapshot();
 });
+
+it('should generate dynamic mocks', async () => {
+    const result = await plugin(testSchema, [], { dynamic: true });
+
+    expect(result).toBeDefined();
+    expect(result).toMatchSnapshot();
+});
```

Comment:
You should probably create another unit test to make sure the `casual.seed` works in unit test

For example, create a new directory like in `tests/circular-mocks`, generate the mocks with a new `create-mocks.ts` file that you run [in jest](https://github.com/ardeois/graphql-codegen-typescript-mock-data/blob/86dbdf1cfd566074637c7119164175e71d3fbd46/jest.config.js#L6) and create a unit test with a call to `casual.seed` like in [here](https://github.com/ardeois/graphql-codegen-typescript-mock-data/blob/main/tests/circular-mocks/spec.ts)

This way, we'll be able to check if the unit test always returns the same values.
You can have a test like:
```typescript
    const a = anA();
    expect(a).toEqual({ foo: 'value' });
```

Where `value` is a string generated by `casual.word` which is called at every test run
If the `casual.seed` works, then the unit test should return the same value every time